start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloVFSClassLoader.java

```diff
@@ -270,6 +270,8 @@ public static FileSystemManager generateVfs() throws FileSystemException {
     vfs.addProvider("tbz2", new org.apache.commons.vfs2.provider.tar.TarFileProvider());
     vfs.addProvider("tgz", new org.apache.commons.vfs2.provider.tar.TarFileProvider());
     vfs.addProvider("bz2", new org.apache.commons.vfs2.provider.bzip2.Bzip2FileProvider());
+    vfs.addProvider("files-cache",
+        new org.apache.commons.vfs2.provider.local.DefaultLocalFileProvider());
```

Comment:
Out of curiosity, why choose this one over the alternatives, such as org.apache.commons.vfs2.cache.SoftRefFilesCache
 (the current default of org.apache.commons.vfs2.impl.DefaultFileSystemManager) or org.apache.commons.vfs2.cache.DefaultFilesCache (the old default for the org.apache.commons.vfs2.impl.DefaultFileSystemManager)?

This isn't a leading question... I'm genuinely curious if there's an advantage of one over another... I don't know the difference between the implementations, or whether it matters.

Reply:
Those classes don't extend  AbstractFileProvider.  Only the classes under the package 'org.apache.commons.vfs2.provider' have providers from what I see.  You could do:
org.apache.commons.vfs2.impl.DefaultFileSystemManager().addProvider(String urlScheme, FileProvider fileprovider) but then you need to know the urlScheme and the FileProvider implementation or program it.  Then we are introducing new code with chances for regression.

I have not been able to replicate this issue.  I am just assuming that the bug reporter has a different system configuration than mine and a solution I propose disposes of the bug for that situation.  I will keep working on this along with some UI issues.

Reply:
Interesting. I wonder if this is just an upstream bug. It seems weird that we'd need to add this provider at all. Perhaps this was fixed in 2.3?